sr_robot_launch/launch/sr_hardware_control_loop.launch

```diff
@@ -51,10 +51,10 @@
   <!-- Set to true to spawn the position controllers for the arm-->
   <arg name="arm_position" default="$(eval not arm_trajectory)"/>
   <arg name="arm_speed_scale" default="0.5"/>
-  <arg if="$(arg hand_ctrl)" name="arm_payload_mass" default="5.00"/>
-  <arg if="$(arg hand_ctrl)" name="arm_payload_cog" default="[0.0, 0.0, 0.12]"/>
-  <arg unless="$(arg hand_ctrl)" name="arm_payload_mass" default="0.0"/>
-  <arg unless="$(arg hand_ctrl)" name="arm_payload_cog" default="[0.0, 0.0, 0.0]"/>
+  <arg if="$(arg hand)" name="arm_payload_mass" default="5.00"/>
```

Comment:
you're changing this but seems like other launchfiles still use hand_ctrl arg callling this file, right?
https://github.com/shadow-robot/sr_interface/blob/296ae6946875de85c3b02fe4679dc7116476d1d6/sr_robot_launch/launch/sr_ur_arm_hand.launch#L56
Have you tested this PR?

Reply:
Yes it was tested. The file you linked pass the hand_ctrl to load or not the controllers. In the low level hardware control loop we only want to set the payload if hand is there and not hand_ctrl as for the arm would be the same regardless if we load the controllers for hand.